azure-monitor-opentelemetry/README.md

```diff
@@ -55,38 +55,20 @@ You can use `configure_azure_monitor` to set up instrumentation for your app to
-| `logging_export_interval_ms`| Specifies the logging export interval in milliseconds. Defaults to 5000. | `OTEL_BLRP_SCHEDULE_DELAY` |
-| `metric_readers` | Specifies the [metric readers][ot_metric_reader] that you would like to use for your metric pipeline. Accepts a list of [metric readers][ot_sdk_python_metric_reader]. | |
-| `views` | Specifies the list of [views][opentelemetry_spec_view] to configure for the metric pipeline. See [here][ot_sdk_python_view_examples] for example usage. | |
-| `sampling_ratio` | Specifies the ratio of distributed tracing telemetry to be [sampled][application_insights_sampling]. Accepted values are in the range [0,1]. Defaults to 1.0, meaning no telemetry is sampled out. | `OTEL_TRACES_SAMPLER_ARG` |
-| `tracing_export_interval_ms`| Specifies the distributed tracing export interval in milliseconds. Defaults to 5000. | `OTEL_BSP_SCHEDULE_DELAY` |
-| `instrumentation_config` | Specifies a dictionary of kwargs that will be applied to instrumentation configuration. You can specify which instrumentation you want to configure by name in the key field and value as a dictionary representing `kwargs` for the corresponding instrumentation. Refer to the `Supported Library` section [above](#officially-supported-instrumentations) for the list of supported library names. | |
-
-Example for use of `instrumentation_config`:
-```python
-...
-configure_azure_monitor(
-    connection_string="<your-connection-string>",
-    instrumentation_config={
-        "flask": {
-            "excluded_urls": "http://localhost:8080/ignore",
-        },
-        "requests": {
-            "excluded_urls": "http://example.com"
-        }
-    }
-)
-...
-```
 
-Take a look at the specific [instrumenation][ot_instrumentations] documentation for available configurations.
+
+You can configure further with [OpenTelemetry environment variables][ot_env_vars] such as:
+| Environment Variable | Description |
+|-------------|----------------------|
+| [OTEL_SERVICE_NAME][opentelemetry_spec_service_name], [OTEL_RESOURCE_ATTRIBUTES][opentelemetry_spec_resource_attributes] | Specifies the OpenTelemetry [resource][opentelemetry_spec_resource] associated with your application. |
+| `OTEL_TRACES_EXPORTER` | If set to `None`, disables collection and export of logging telemetry. |
+| `OTEL_METRICS_EXPORTER` | If set to `None`, disables collection and export of metric telemetry. |
+| `OTEL_LOGS_EXPORTER` | If set to `None`, disables collection and export of distributed tracing telemetry. |
+| `OTEL_BLRP_SCHEDULE_DELAY` | Specifies the logging export interval in milliseconds. Defaults to 5000. |
+| `OTEL_TRACES_SAMPLER_ARG` | Specifies the ratio of distributed tracing telemetry to be [sampled][application_insights_sampling]. Accepted values are in the range [0,1]. Defaults to 1.0, meaning no telemetry is sampled out. |
```

Comment:
Nit: Switch the ordering of the last two so `OTEL_x_SCHEDULE_DELAY` variables are grouped together.

Reply:
Done.